recipe/meta.yaml

```diff
@@ -108,78 +121,57 @@ outputs:
-        - libtiff                            # [not win]
-        - libblas
-        - liblapack
+        - icu
+        - pcre2
+        - tk
+        - zlib
+        - gsl
+        - libtiff
+        - libblas                            # [not win]
+        - liblapack                          # [not win]
         # sed is in host env because its path is recorded in R's shell wrapper
         - sed                                # [linux]
       run:
         - _r-mutex 1.* anacondar_1
         # Needed by R CMD javareconf
         - make                               # [not win]
-        - gcc_impl_{{ target_platform }}  >={{ c_compiler_version|default("0") }}    # [linux]
-        - gxx_impl_{{ target_platform }}  >={{ cxx_compiler_version|default("0") }}  # [linux]
-        - gfortran_impl_{{ target_platform }}  # [linux]
+        - gcc_impl_{{ target_platform }}  >={{ c_compiler_version|default("0") }}    # [linux or win]
+        - gxx_impl_{{ target_platform }}  >={{ cxx_compiler_version|default("0") }}  # [linux or win]
+        - gfortran_impl_{{ target_platform }}  # [linux or win]
         - {{ c_compiler|default("blah") }}_{{ target_platform }} >={{ c_compiler_version|default("0") }}      # [osx]
         - {{ cxx_compiler|default("blah") }}_{{ target_platform }} >={{ cxx_compiler_version|default("0") }}  # [osx]
         - {{ compiler('fortran') }}  # [osx]
-        - curl                                 # [not win]
-        - {{ native }}bzip2                    # [win]
-        - {{ native }}libjpeg-turbo            # [win]
-        - {{ native }}libiconv                 # [win]
+        - {{ pin_compatible('llvm-openmp', max_pin=None) }}      # [osx]
+        - curl
+        - libuuid                              # [linux]
         - xorg-libxt                           # [linux]
-        - {{ native }}gmp                      # [win]
-        - {{ native }}fftw                     # [win]
-        - {{ native }}xz                       # [win]
-        - {{ native }}mpfr                     # [win]
-        - {{ native }}libsndfile               # [win]
-        - {{ native }}bwidget
-        - {{ native }}tktable
-        - {{ native }}libpng                   # [win]
-        - {{ native }}pcre2                    # [win]
-        - {{ native }}tk                       # [win]
-        - {{ native }}zlib                     # [win]
-        - {{ native }}libtiff                  # [win]
-        - sed                                # [linux]
+        - bwidget
+        - tktable
+        - tk
+        - sed                                  # [linux]
 
     test:
       commands:
         - R -h
         - R --version
-        - R -e "print(.Platform\$dynlib.ext)"
-        - R -e "library('tcltk')"            # [linux or osx]
-        - R -e \"library('tcltk')\""         # [win]
+        - R -e "print(.Platform\$dynlib.ext)"    # [not win]
+        - R -e \"print(.Platform$dynlib.ext)\"   # [win]
```

Comment:
Logs are showing this and similarly-escaped expressions are getting eval'd as string literals in the tests. On the other hand, line 197 that has no selector or escapes is correctly evaluated. I think we can just stop escaping and drop the selectors for common evaluation/library loading tests.

Reply:
Ah, good catch!
It's a bit disconcerting that it previously handled the doubly-quoted strings differently... (checked the latest 4.1.x build and there it resolved the inner marks before evaluating)
The first addition of those doubly-quoting seemed to be in
https://github.com/conda-forge/r-base-feedstock/commit/23f725418a405b4065ecfd93b9119f7309301199#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR148
and then
https://github.com/conda-forge/r-base-feedstock/commit/3af0aae34b80d8103fff994d626417fef4304e12
with the former commit having the `\"` for YAML string quoting whereas the latter apparently failed to remove the trailing one of the quotes pair.
I.e., it rather looks like an error that happened to work? Really odd.